docker-compose.yml

```
@@ -60,9 +59,11 @@ services:
     environment:
       DATABASE_URL: 'postgis://postgres:@postgres/lametro'
       SHARED_DB: "True"
-    command: sh -c 'pupa update --rpm=600 lametro people && pupa update --rpm=600 lametro bills window=30 && pupa update --rpm=600 lametro events'
+      OCD_DIVISION_CSV: '/app/lametro_divisions.csv'
+    command: sh -c 'pupa update --rpm=600 lametro people && pupa update --rpm=600 lametro bills window=14 && pupa update --rpm=600 lametro events'
```

Last thing! Let's narrow the window of the event scrape so startup is faster. Then this can come in.

```suggestion
    command: sh -c 'pupa update --rpm=600 lametro people && pupa update --rpm=600 lametro bills window=14 && pupa update --rpm=600 lametro events window=14'
```